numba/cgutils.py

```diff
@@ -643,19 +651,14 @@ def is_neg_int(builder, val):
     return builder.icmp(lc.ICMP_SLT, val, get_null_value(val.type))
 
 
-def inbound_gep(builder, ptr, *inds):
-    idx = []
-    for i in inds:
-        if isinstance(i, int):
-            ind = Constant.int(Type.int(32), i)
-        else:
-            ind = i
-        idx.append(ind)
-    return builder.gep(ptr, idx, inbounds=True)
+def gep_inbounds(builder, ptr, *inds, **kws):
+    return gep(builder, ptr, *inds, inbounds=True, **kws)
 
 
 def gep(builder, ptr, *inds, **kws):
```

Comment:
@pitrou Can you give a short description of this function? Maybe as a docstring.